BDKSwiftExampleWallet/View/OnboardingView.swift

```diff
@@ -122,24 +122,35 @@ struct OnboardingView: View {
                     .opacity(animateContent ? 1 : 0)
                     .animation(.easeOut(duration: 0.5).delay(1.5), value: animateContent)
 
-                    Picker("Esplora Server", selection: $viewModel.selectedURL) {
-                        ForEach(viewModel.availableURLs, id: \.self) { url in
-                            Text(
-                                url.replacingOccurrences(
-                                    of: "https://",
-                                    with: ""
-                                ).replacingOccurrences(
-                                    of: "http://",
-                                    with: ""
-                                )
-                            )
-                            .tag(url)
-                        }
+                    Picker("Sync type", selection: $viewModel.syncMode) {
+                        Text("Esplora Server").tag(SyncMode.esplora)
+                        Text("Kyoto").tag(SyncMode.kyoto)
```

Comment:
![Simulator Screen Recording - iPhone 16 Pro - 2025-05-28 at 11 32 08](https://github.com/user-attachments/assets/a5ad2b7d-8b6b-49e1-aba5-9b0277df89c4)

I'm seeing this (bug?) when I load OnboardingView for the first time, where I think maybe its supposed to say "Esplora Server" as the default here? Maybe that's not the issue, but attached is a gif of what I'm seeing